doc/classes/TouchScreenButton.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TouchScreenButton" inherits="Node2D" version="4.0">
 	<brief_description>
-		Button for touch screen devices.
+		Button for touch screen devices for gameplay use.
 	</brief_description>
 	<description>
-		Button for touch screen devices. You can set it to be visible on all screens, or only on touch devices.
+		Button for touch screen devices for gameplay use.
```

Comment:
The description shouldn't include a copy of the brief description. Try to use sentences that make the class's intended purpose as clear as possible. Suggestion:

> TouchScreenButton allows you to create on-screen buttons for touch devices. It's intended for gameplay controls, like a button the player can press to jump.